server/tools/attestation-service-ctl/src/attestation.rs

```diff
@@ -0,0 +1,24 @@
+use anyhow::{anyhow, Result};
+use std::fs;
+use std::path::Path;
+
+use crate::attestation_api::attestation_service_client::AttestationServiceClient;
+use crate::attestation_api::{AttestationRequest, AttestationResponse};
+
+pub async fn attestation_cmd(evidence_path: &Path, socket: &str) -> Result<()> {
```

Comment:
The value passed to this function isn't a socket, it's an IPv4 address fwics, so this parameter name is confusing.